nobl9/resource_slo_test.go

```diff
@@ -1628,7 +1628,7 @@ resource "nobl9_slo" ":name" {
 
   }
 
-  attachments {
+  attachment {
```

Comment:
since we are supporting two versions, can we please test both? :pray: 
and maybe a test that checks what happens when we have both?

Reply:
Sure no prob